sonaranalyzer-dotnet/its/expected/Ember-MM/Ember.Plugins-{9496C697-5AFD-4813-AEDC-AF33FACEADF0}-S4058.json

```diff
@@ -0,0 +1,69 @@
+{
+"issues":  [
+{
+"id":  "S4058",
+"message":  "Change this call to 'lTitle.EndsWith' to an overload that accepts a 'StringComparison' as a parameter.",
```

Comment:
I am mentioning the full expression of the invocation, maybe it should be reduced, WDYT?